application/models/user.go

```diff
@@ -146,6 +147,29 @@ func (u *User) CreateAccessToken(org Organization, clientID string) (string, int
 	return token, expireAt.UTC().Unix(), nil
 }
 
+// CreateOrglessAccessToken - Create and store new UserAccessToken with no associated UserOrg
+func (u *User) CreateOrglessAccessToken(clientID string) (string, int64, error) {
```

Comment:
What if you made the `org` parameter of `CreateAccessToken` a pointer or test for `ID == 0`? These two functions are nearly identical.

Reply:
:+1: